### PR TITLE
Testing updates

### PR DESCRIPTION
**Pull request by nathanaelren**
- updated gitigore to ignore the massive robot recording files
- added a brief guide on testing migration w/ foxglove

### CODE REVIEW

**Review by maxwellmlin — CHANGES_REQUESTED** [review]
Awesome! Thanks for setting this up. A few minor suggestions:

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,15 @@ __pycache__
 **/install/*
 **/log/*
 
+# Robot playback folders
+**/ros2_mcap_test/*
```
[the remark this answers is not in the document]
Comment (nathanaelren) [reply .gitignore:9]: Yeah this definitely makes more sense, adding a file named `bag_files` and ignoring all subfolders

**Comment by nathanaelren** [comment]
DukeRobotics/wiki#3

File: ros2_migration/README.md
Comment (maxwellmlin) [inline ros2_migration/README.md:1]: Note that we'll probably delete the `ros2_migration` folder once we're done with migration.

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,15 @@ __pycache__
 **/install/*
 **/log/*
 
+# All playback subfolders
+**/bag_files/*
```
Comment (maxwellmlin) [inline .gitignore:9]: I think should be `bag_files/**`. The `bag_files` directory should only appear in the root level and everything inside should be ignored, not just 1-level deep.

**Review by maxwellmlin — APPROVED** [review]
Just address comment about gitignore then LGTM!

**Pull request merged**